config/config.file.extended.hocon

```diff
@@ -31,6 +31,14 @@
       "file": "/var/bad"
       "maxBytes": 1000000
     }
+
+    # Incomplete events output
+    "incomplete": {
```

Comment:
I assume simply omitting this key disables the feature yeah?

Reply:
Yes! Similarly too pii events

Reply:
Super. Love it.